.circleci/config.yml

```
@@ -7,6 +7,9 @@ parameters:
   l1_mainnet_rpc_url:
     type: string
     default: "https://ci-mainnet-l1.optimism.io"
+  l1_mainnet_archive_rpc_url:
+    type: string
+    default: "https://ci-mainnet-l1-archive.optimism.io"
```

Would this work, and simplify the config?
```suggestion
  l1_mainnet_rpc_url:
    type: string
    default: "https://ci-mainnet-l1-archive.optimism.io"
```